packages/offline-integration/.eslintrc.json

```diff
@@ -0,0 +1,24 @@
+{
+    "extends": ["../../.eslintrc.json"],
+    "ignorePatterns": ["!**/*"],
+    "overrides": [
+        {
+            "files": ["*.ts", "*.tsx", "*.js", "*.jsx"],
+            "rules": {
```

Comment:
Do we need to ignore these rules? The strategy files seem pretty simple.

Reply:
Yes, make sense. I removed all overrides here.

Reply:
Do we need the override for the spec files? Since there is an issue with jest types that imply that `expect(something)` returns type of `any`.

Reply:
I restored three rules.
How could CI check pass without turning off any rules in the previous commit😂

Reply:
From what I know they are lint issues and we don't run lint for build, but only on release 🤦

Reply:
> From what I know they are lint issues and we don't run lint for build, but only on release 🤦

Are you sure that's the case? I think we run the linter on every PR? https://github.com/bigcommerce/checkout-sdk-js/blob/f0cb31f27d7d4487bf713de9acd638b95734dcce/.circleci/config.yml#L33

Reply:
Yes we do on another look, which is more confusing as to how come the build passed? Might test this in a next PR to validate.